src/main/java/org/kohsuke/github/GHHooks.java

```diff
@@ -88,6 +88,47 @@ public GHHook createHook(String name, Map<String, String> config, Collection<GHE
             return wrap(hook);
         }
 
+        /**
+         * Update hook gh hook.
+         *
+         * @param id
+         *            the ID
+         * @param name
+         *            the name
+         * @param config
+         *            the config
+         * @param events
+         *            the events
+         * @param active
+         *            the active
+         * @return the gh hook
+         * @throws IOException
+         *             the io exception
+         */
+        public GHHook updateHook(long id,
+                String name,
+                Map<String, String> config,
+                Collection<GHEvent> events,
+                boolean active) throws IOException {
+            List<String> ea = null;
+            if (events != null) {
+                ea = new ArrayList<String>();
+                for (GHEvent e : events)
```

Comment:
:warning: 'for' construct must use '{}'s. (#QR508:116#)-CHECKSTYLE_NeedBracesCheck